src/radical/pilot/agent/bootstrap_0.sh

```diff
@@ -1058,19 +1058,8 @@ virtenv_create()
     # of the RADICAL stack
     for dep in $VIRTENV_RADICAL_DEPS
     do
-        # NOTE: we have to make sure not to use wheels on titan
-        hostname | grep titan 2&>1 >/dev/null
-        if test "$?" = 1
-        then
-            # this is titan
-          # wheeled="--no-use-wheel"
-            wheeled="--no-binary :all:"
-        else
-            wheeled=""
-        fi
-
         run_cmd "install $dep" \
-                "$PIP install $wheeled $dep" \
+                "$PIP install $dep" \
              || echo "Couldn't install $dep! Lets see how far we get ..."
     done
 
```

Comment:
This is not needed on titan anymore and in fact breaks there.  Both titan and summit can use the default code path here.